app/assets/javascripts/discourse/config/optional-features.json.js

```diff
@@ -3,5 +3,5 @@ module.exports = {
   "default-async-observers": true,
   "jquery-integration": false,
   "template-only-glimmer-components": true,
-  "no-implicit-route-model": true,
+  "no-implicit-route-model": false,
```

Comment:
I think we still have plugins/theme components that would fail if this was disabled?

Reply:
(or maybe, hopefully, those are model-less routes that don't use it at all)

Reply:
Okay, a cursory search didn't reveal any route+controller combos that would fail 👍

Reply:
> though I would have expected there to be issues reported by now if any stuff in the wild was crashing due to this since this flag was true a few months ago.

Yeah, this means nothing was offending 🚀